internal/provisioner/plain/README.md

```diff
@@ -135,23 +135,30 @@ resources referenced by the bundle are present on the cluster.
 ### Setup
 
 To experiment with the `plain` provisioner locally, first setup a local cluster, or simply
-have [kind](https://kind.sigs.k8s.io/) installed locally.
+have [kind](https://kind.sigs.k8s.io/) installed and a kind cluster running locally.
```

Comment:
This probably changes once #365 closes since it'll be creating a local copy of the kind binary.